packages/next-app/src/app/(components)/RootLayout.tsx

```diff
@@ -21,6 +21,11 @@ export function RootLayout({ children }: { children: React.ReactNode }) {
   const { isConnected } = useAccount();
   const router = useRouter();
 
+  const RainbowKitProviderDynamic = dynamic(
+    async () => (await import("@rainbow-me/rainbowkit")).RainbowKitProvider,
+    { ssr: false }
+  );
```

Comment:
Curious about what motivated this change 🤔

Reply:
I used dynamic here to clean up some uncatched errors of the type: 
"Hydration failed because the initial UI does not match what was rendered on the server"
For now, works fine. 
But I'm curious about your troubles with _dynamic_ and whether this could affect us here.

cc @devjoaov

Reply:
np! I'd be ok ignoring the hydration errors, but glad you took it to heart 🙇